pkgs/by-name/be/better-control/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  gtk3,
+  networkmanager,
+  bluez,
+  pipewire,
+  brightnessctl,
+  power-profiles-daemon,
+  gammastep,
+  libpulseaudio,
+  pulseaudio,
+  desktop-file-utils,
+  wrapGAppsHook4,
+  gobject-introspection,
+  usbguard,
+  upower,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "better-control";
+  version = "v6.11.4";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "quantumvoid0";
+    repo = "better-control";
+    tag = version;
+    hash = "sha256-i+r0iPeSG2a8P2oDvm7QS+Y1ok117K6gnC7EVJuKgIs=";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    wrapGAppsHook4
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libpulseaudio
+  ];
+
+  dependencies =
+    [
+      networkmanager
+      bluez
+      pipewire
+      brightnessctl
+      power-profiles-daemon
+      gammastep
+      pulseaudio
+      usbguard
+      upower
```

Comment:
Non-python dependencies should be in `buildInputs`, also please set `build-system` explicitly.